libraries/testkit/server.py

```diff
@@ -0,0 +1,139 @@
+import base64
+import json
+import time
+
+import requests
+
+from couchbase.bucket import Bucket
+from libraries.provision.ansible_runner import AnsibleRunner
+
+from keywords.utils import log_info
+from keywords.utils import log_error
+from keywords.couchbaseserver import create_internal_rbac_bucket_user
+from keywords.couchbaseserver import delete_internal_rbac_bucket_user
+from keywords.couchbaseserver import get_server_version
+
+
+class Server:
```

Comment:
Very good catch. I removed it again. I'm guessing it was a product of an old merge.